packages/ui-core/shared/src/lib/timesheet/gauzy-filters/gauzy-filters.component.scss

```
@@ -181,14 +181,41 @@ nb-select {
         }
 
+        // The activity-level filter is an `nbButton` standing in a row of `nb-select`
+        // triggers, so it arrives carrying button typography (700 weight, 0.875rem) and,
+        // because it is `outline`, a real 1px border — while the selects beside it draw
+        // `border: none` plus the shared inset hairline from `nb-select-overrides`. Side
+        // by side that reads as one control being styled by a different system, so the
+        // button restates the select's own tokens instead of Nebular's button scale.
         .activity-level-filter {
-            padding: 7px 7px 7px 11px !important;
-            border: 1px solid nb-theme(select-outline-basic-border-color) !important;
+            // Matches the select trigger's box: the same 3px block inset as
```

Stylelint (17.14.1)

[error] Unknown word Matches (CssSyntaxError)

(CssSyntaxError)